src/layer/Tooltip.js

```diff
@@ -411,9 +411,18 @@ Layer.include({
 
 
 	_openTooltip(e) {
-		if (!this._tooltip || !this._map || (this._map.dragging && this._map.dragging.moving())) {
+		if (!this._tooltip || !this._map) {
 			return;
 		}
+
+		// If the map is moving, we will show the tooltip after it's done.
+		if (this._map.dragging && this._map.dragging.moving()) {
```

Comment:
I think we can make this a bit shorter using optional chaining.
```suggestion
		if (this._map.dragging?.moving()) {
```

Reply:
Good idea.  Done.